src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/StandardDirectionOrderDJ.java

```diff
@@ -785,6 +786,8 @@ private CallbackResponse generateSDONotifications(CallbackParams callbackParams)
                 caseDataBuilder.hmcEaCourtLocation(featureToggleService.isLocationWhiteListedForCaseProgression(
                     caseData.getCaseManagementLocation().getBaseLocation()) ? YES : NO);
             }
+        } else if (isLipCaseWithProgressionEnabledAndCourtWhiteListed(caseData)) {
```

Comment:
Same here, I think - now if a case is also it's part of rollout, it will never come here.

Reply:
Done